subprojects/core/src/main/resources/org/gradle/initialization/removed-types.txt

```diff
@@ -25,3 +25,6 @@ org.gradle.api.artifacts.transform.VariantTransform
 org.gradle.api.tasks.incremental.IncrementalTaskInputs
 org.gradle.workers.WorkerConfiguration
 org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep$Operation$Details
+org.gradle.api.plugins.Convention
```

Comment:
I'm extending the use of this concept a "bit".

Maybe separate it from removed-types.txt into something else?